src/templates/event.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+import { Box, Heading, Text } from 'grommet';
+
+import { Content, Layout, Markdown, SEO, Link } from '../components';
+import { useSiteMetadata } from '../hooks/use-site-metadata';
+
+// Remove padding or margin from first markdown element.
+// This allows the heading and content to have the same gap.
+const MarkdownLayout = styled(Markdown)`
+  & > *:first-child {
+    margin-top: 0;
+    padding-top: 0;
+  }
+`;
+const handleDate = (dateStart, dateEnd, dateFormat) => {
```

Comment:
It looks fine although I'd recommend getting in the habit of not duplicating as much code (aka DRY). If you end up with a bug or just want to the Text to have different props you have to find and fix multiple places. One way is to do something more like:
`
const handleDate = (dateStart, dateEnd, dateFormat) => {
  let dateString = dateFormat.format(new Date(dateStart);
  if (dateStart !== dateEnd) {
    dateString = `${dateString} - ${dateFormat.format(new Date(dateEnd))}`;
  return <Text size="large">{dateString}</Text>;
};
`

Reply:
gotcha, changes made.